api-specificatie/zrc/openapi.yaml

```diff
@@ -1,83 +1,504 @@
             application/json:
               schema:
-                $ref: '#/components/schemas/Zaak'
-        422:
-          description: Opgegeven zaak informatie kan niet verwerkt worden
+                $ref: '#/components/schemas/KlantContact'
+      tags:
+        - klantcontacten
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this klantcontact.
+        required: true
+        schema:
+          type: integer
+  /rollen:
+    post:
+      operationId: rol_create
+      description: Koppel een BETROKKENE aan een ZAAK.
+      responses:
+        '201':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Fout'
-        default:
-          description: Onverwachte fout tijdens opvoeren zaak
+                $ref: '#/components/schemas/Rol'
+      tags:
+        - rollen
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Rol'
+        required: true
+    parameters: []
+  '/rollen/{id}':
+    get:
+      operationId: rol_read
+      description: ''
+      responses:
+        '200':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Fout'
-    parameters: []
-  /zaakinformatieobjecten:
+                $ref: '#/components/schemas/Rol'
+      tags:
+        - rollen
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this Rol.
+        required: true
+        schema:
+          type: integer
+  /statussen:
     post:
-      summary: Voeg een informatieobject toe aan de zaak
-      operationId: zaakinformatieobject_create
-      description: Voeg een informatieobject toe aan een zaak
+      operationId: status_create
+      description: ''
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status'
+      tags:
+        - statussen
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/ZaakInformatieObject'
+              $ref: '#/components/schemas/Status'
+        required: true
+    parameters: []
+  '/statussen/{id}':
+    get:
+      operationId: status_read
+      description: ''
       responses:
-        201:
-          description: ZaakInformatieObject is succesvol aangemaakt.
+        '200':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schema/ZaakInformatieObject'
-        422:
-          description: Opgegeven zaakinformatieobject kan niet verwerkt worden
+                $ref: '#/components/schemas/Status'
+      tags:
+        - statussen
+    parameters:
```

Comment:
In het RGBZ2 heeft een status  geen identificatie. 

De ID is deze API is dus puur een technisch ID, geen gegeven uit het RGBZ2. Zou dit iets in een UGM moeten worden? @zomwoof